docs/user-guide/secrets.md

```diff
@@ -162,15 +161,21 @@ See another example of creating a secret and a pod that consumes that secret in
 
 Use of imagePullSecrets is described in the [images documentation](images.md#specifying-imagepullsecrets-on-a-pod)
 
-### Automatic use of Manually Created Secrets
+### Arranging for imagePullSecrets to be Automatically Attached
+
+You can manually create an imagePullSecret, and reference it from
```

Comment:
I think this would read a bit better if split into two sentences:

"You can manually create an `imagePullSecret` and reference it from a `serviceAccount`. Any pods created with that `serviceAccount`, or that default to use that `serviceAccount`, will use that `imagePullSecret`."